scripts/build/termux_step_get_dependencies.sh

```diff
@@ -45,15 +47,15 @@ termux_step_get_dependencies() {
 				build_dependency=true
 			else
 				if termux_package__is_package_version_built "$PKG" "$DEP_VERSION"; then
-					[ ! "$TERMUX_QUIET_BUILD" = true ] && echo "Skipping already built dependency $PKG$(test ${TERMUX_WITHOUT_DEPVERSION_BINDING} = false && echo "@$DEP_VERSION")"
+					[ ! "$TERMUX_QUIET_BUILD" = true ] && echo "Skipping already built dependency $pkg_versioned"
```

Comment:
*Checks notes*
So...
```suggestion
					[[ "$TERMUX_QUIET_BUILD" != 'true' ]] && echo "Skipping already built dependency $pkg_versioned"
```